src/taxman.py

```diff
@@ -475,6 +477,54 @@ def _evaluate_taxation_GERMANY(self, op: tr.Operation) -> None:
                 )
                 self.tax_report_entries.append(report_entry)
 
+        elif isinstance(op, tr.MarginFee):
+            # Fees for margin trading
+            self.remove_from_balance(op)
+            if in_tax_year(op):
+                taxation_type = "Kapitaleinkünfte aus Margin Trading"
```

Comment:
Does this category really exists? I thought that these are normal "Kapitaleinkünfte". Please adjust or clarify.